### DISCUSSION TITLE
Global settings | Chulapa

### DISCUSSION THREAD

**Discussion by giscus** [dieghernan/chulapa#6] [General]
# Global settings | Chulapa

Learn how to set up your new site - Setting up your new site

https://dieghernan.github.io/chulapa/docs/02-config

**Comment by dieghernan** [comment]
My first comment with giscus!

**Reply by dieghernan** [reply]
We can use Markdown:

See **bold** and _italic_.

 A list:

* One
* Two

# A header

## A second header

A [link](https://dieghernan.github.io/chulapa/docs/01-install)

An Image
![](https://dieghernan.github.io/chulapa/assets/img/site/chulapa-madrid.jpg)
